test/FastTests/Corax/Bugs/FacetIndexingRepro.cs

```diff
@@ -147,22 +147,19 @@ public unsafe void ShouldNotCorrupt()
                     continue;
                 }
 
-                int len = br.Read7BitEncodedInt();
-                var buffer = br.ReadBytes(len);
                 using var indexEntryBuilder = iw.Index(id);
-                fixed (byte* b = buffer)
                 {
-                    var reader = new IndexEntryReader(b, buffer.Length);
                     for (int i = 0; i < fieldsCount; i++)
                     {
-                        var fieldReader = reader.GetFieldReaderFor(i);
-                        fieldReader.Read(out Span<byte> s);
-                        indexEntryBuilder.Write(i, s);
+                       int len = br.Read7BitEncodedInt();
```

Comment:
What are we writing and why.

Reply:
I ported a failing test that embedded the raw index entries, to remove that, I changed the format of the file and now we are reading it this way